microreactors/gcmr/core/Multiphysics/null_transient/MP_Griffin_trN.i

```diff
@@ -0,0 +1,200 @@
+fuel_blocks = '400 4000 40000 401 4001 40001'
+he_channel_blocks = '200 201'
+mod_blocks = '100 101'
+poison_blocks = '19000 29000 39000 49000 59000 19003 29003 39003 49003 59003 19900 29900 39900 49900 59900 19903 29903 39903 49903 59903'
+ref_blocks = '1000 1003 250 600 602 1777 1773'
+he_void_blocks = '300 301 604'
+cd_poison_blocks = '603'
+fecral_blocks = '103'
+cr_blocks = '102'
+small_parts_blocks = '${cd_poison_blocks} ${fecral_blocks} ${cr_blocks}'
+monolith_blocks = '10'
+
+non_he_channel_blocks = '${fuel_blocks} ${mod_blocks} ${poison_blocks} ${ref_blocks} ${he_void_blocks} ${small_parts_blocks} ${monolith_blocks}'
+
+[Mesh]
```

Comment:
is this the same mesh as the steady state?
you could keep the procedure for making a new mesh only in the steady input and point to it here